### PR TITLE
cluster: delete lot's of stuff

### PR DESCRIPTION
**Pull request by mikedanese**
let me know if any of this is too aggressive. 

see #49213

```release-note
Remove unmaintained kube-registry-proxy support from gce kube-up.
```

### CODE REVIEW

File: cluster/common.sh [hunk 1]
```diff
@@ -293,28 +273,6 @@ function load-or-gen-kube-bearertoken() {
...
-# Assumed vars:
-#   KUBECONFIG  # if unset, defaults to global
-#   KUBE_CONTEXT  # if unset, defaults to current-context
-#
-# Vars set:
-#   KUBE_MASTER_URL
-#
-# KUBE_MASTER_URL will be empty if no current-context is set, or the
-# current-context user does not exist or contain a server entry.
-function detect-master-from-kubeconfig() {
```
Comment (roberthbailey) [inline cluster/common.sh]: It looks like this is used somewhere in the test framework:

```
W0120 06:24:49.133] ./hack/ginkgo-e2e.sh: line 53: detect-master-from-kubeconfig: command not found
W0120 06:24:49.136] !!! Error in ./hack/ginkgo-e2e.sh:53
W0120 06:24:49.136]   Error in ./hack/ginkgo-e2e.sh:53. 'detect-master-from-kubeconfig' exited with status 127
```

Comment (roberthbailey) [reply cluster/common.sh]: https://github.com/kubernetes/kubernetes/blob/master/hack/ginkgo-e2e.sh#L53

**Comment by zouyee** [comment]
Since registry example is to be deleted,I will close the PR about the registry.
https://github.com/kubernetes/kubernetes/pull/57928

**Comment by zouyee** [comment]
Would you like to add a TODO list for tracking process flow about clean work?

**Comment by justinsb** [comment]
e2e should be fixed by https://github.com/kubernetes/kubernetes/pull/57896

**Comment by justinsb** [comment]
/retest

quota limits on vpcs in new account

**Comment by justinsb** [comment]
Opened https://github.com/kubernetes/kubernetes/issues/58578 to track the NFS failures.

**Pull request merged**